code/src/App.js

```diff
@@ -1,9 +1,14 @@
-import React from 'react'
+import React from 'react';
+import Footer from "components/Footer";
+import Form from 'components/Form';
+
+
 
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+  return <main>
+
+<Form />
+<Footer />
+
```

Comment:
Nice job for this last week, who does not love Macarons :) 
Nice with all the color on the site, give it a warm feeling